druid/src/app.rs

```diff
@@ -70,7 +70,7 @@ impl<T: Data + 'static> AppLauncher<T> {
     /// Set the [`AppDelegate`].
     ///
     /// [`AppDelegate`]: struct.AppDelegate.html
-    pub fn delegate(mut self, delegate: AppDelegate<T>) -> Self {
+    pub fn delegate(mut self, delegate: Box<dyn AppDelegate<T>>) -> Self {
```

Comment:
I would make this argument be, `impl AppDelegate<T>`, and then we can box inside the function; this just saves the caller from the hassle of having to box before they call.

(you might also need a `+ 'static` after the type)